Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM nginx
```

Comment:
Actually, if this is only for local use you don't need an image at all. You just need a webapp.conf and `owp nginx`